_source/_posts/2021-08-23-free-virtual-developer-day-auth-for-all.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: blog_post
+title: "Developer Day 2021:  Authentication, Authorization, and Celebrating Our Diverse Developer Community"
+Author: ali-spivak
+by: advocate 
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "Announcing Developer Day 2021: Auth for All with Okta and Auth0. Check out our speaker lineup, and register now. "
+tags: [identity, authorization, authentication, developer community]
+tweets:
+- ""
+- ""
+- ""
+Image: blog/
+type: awareness|conversion
+---
+
+# Developer Day 2021: A Free Virtual Event to Celebrate the Future of Identity 
+
+
+It’s been a big year for Okta and Auth0. Our two companies have officially joined forces over the past few months, and we are excited about the chance to shape the future of identity together. It’s great to celebrate our combined developer communities at the first Developer Day event. 
```

Comment:
```suggestion
---

It’s been a big year for Okta and Auth0. Our two companies have officially joined forces over the past few months, and we are excited about the chance to shape the future of identity together. It’s great to celebrate our combined developer communities at the first Developer Day event. 
```

The `H1`/ top level heading gets added automatically from the `title` field above.  Basically, everything in the post needs to be an `h2` or lower